frontend/src/App.vue

```diff
@@ -17,7 +17,7 @@
     </header>
     <div class="">
       <particles-bg type="custom" :config="config" :bg="true" />
-      <router-view />
+      <component :is="$store.state.session_id ? 'DashboardLayout' : 'AuthLayoutGDD'" />
```

Comment:
Is this new structure really an improvement? This limits the layout to two and decouples it from the router.
Can we do something like this?

```
import myLayout1 from [...]

[...]

  {
    path: '/profile',
    component: () => import('../views/Pages/UserProfileCard.vue'),
    layout: myLayout1
  },
```

Reply:
This is nuxt. I did not find a way to do this with only vue-router so far. But I agree, this would be the best solution.

Reply:
@ogerly  achieved this by using faked children. But we do not have children.

Reply:
We could use a computed property with a switch to implement more than two layouts

Reply:
So merge as you suggested or refactor in some way? nuxt?